src/components/ConfirmDelete/ConfirmDelete.tsx

```diff
@@ -32,7 +32,8 @@ export default class ConfirmDelete extends React.PureComponent<Props, State> {
         <Confirm
           size={size}
           open={isOpen}
-          content={t('confirm_delete.content', { name })}
+          header={t('confirm_delete.content', { name })}
+          content={t('confirm_delete.subcontent', { name })}
```

Comment:
this adds the second line adding the phrase that the action is not reversible

<img width="990" alt="image" src="https://user-images.githubusercontent.com/8763687/208988550-8400e6d8-aa4a-4144-b763-0449530b08eb.png">